### ISSUE TITLE
Dataset Split Question

### ISSUE THREAD

**Issue by linyunooo** [ShenhanQian/GaussianAvatars#126]
Hello,

I have a question regarding the dataset split.

While inspecting the dataset, I noticed that camera 8 appears in the transforms_test.json file, but it is not included in transforms_train.json.

From my understanding, the completed test experiment is supposed to be self-reenactment. In a self-reenactment setting, shouldn't only the facial expressions be changed while keeping the camera viewpoints consistent with the training set? If camera 8 is excluded from training and only appears in the test set, wouldn't this introduce a novel-view component in addition to the expression change?

Could you please clarify whether this split is intentional and explain the rationale behind it?

Thank you for your time and help.

<img width="666" height="109" alt="Image" src="https://github.com/user-attachments/assets/359a1840-d8e6-4d3f-b6d1-981c6f59dfdc" />

<img width="620" height="103" alt="Image" src="https://github.com/user-attachments/assets/e8924dba-a734-4c52-aeb4-30c127a46c61" />
